app/Core/Router.php

```diff
@@ -29,4 +51,83 @@ private function __construct() {
     public static function getInstance() : Router {
         return self::$routerInstance ?? new Router();
     }
+
+    /**
+     *  Get Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function get($url, $action){
+        $this->addRoute(self::GET, $url, $action);
+    }   
+
+    /**
+     *  Post Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function post($url, $action){
+        $this->addRoute(self::POST, $url, $action);
+    }   
+
+
+    /**
+     *  Delete Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function delete($url, $action){
+        $this->addRoute(self::DELETE, $url, $action);
+    }   
+
+    /**
+     *  Patch Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function patch($url, $action){
+        $this->addRoute(self::PATCH, $url, $action);
+    }
+
+    /**
+     * Adds newly registered route to routes array
+     *
+     * @param string $request_type
+     * @param string url
+     * @param string action
+     */
+    protected function addRoute($request_type=self::GET, $url, $action){
```

Comment:
This line would not work, the default argument would throw an error.
Check here: http://php.net/manual/en/functions.arguments.php for more infor. Look under incorrect usage of default function argument.